{
    "type": "bundle",
    "id": "bundle--2020c8ed-8e4f-489d-b6ce-b922bd4113f4",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--02dccb0c-5d94-4bdb-8271-f4fddfd64eb3",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-09-14T01:06:11.004023Z",
            "modified": "2024-09-14T01:06:11.004023Z",
            "name": "CVE-2024-29779",
            "description": "there is a possible escalation of privilege due to an unusual root cause. This could lead to local escalation of privilege with no additional execution privileges needed. User interaction is not needed for exploitation.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-29779"
                }
            ]
        }
    ]
}